server/lib/twitch.js

```diff
@@ -0,0 +1,64 @@
+const fetch = require('node-fetch');
+const WebSocket = require('ws');
+
+//ANDREW PLEASE
+function openSocketConnectionWithChannelId(channelId, oauth_token, callBack) {
+    ws = new WebSocket('wss://pubsub-edge.twitch.tv');
+
+    function heartbeat() {
+        message = {
+            type: 'PING',
+        };
+        console.log('SENT: ' + JSON.stringify(message));
+        ws.send(JSON.stringify(message));
+    }
+    var heartbeatInterval = 1000 * 60; //ms between PING's
+    var reconnectInterval = 1000 * 3; //ms to wait before reconnect
+    var heartbeatHandle;
+    ws.onopen = function (event) {
+        console.log('INFO: Socket Opened');
+        heartbeat();
+        heartbeatHandle = setInterval(heartbeat, heartbeatInterval);
+
+        // When the connection is opened, fire a LISTEN event to the WSS
+        let listenEvent = {
+            type: 'LISTEN',
+            nonce: 'abc123',
```

Comment:
I'll make an issue for this. not that important, but I think we can do better here. Not worth holding up this PR though